src/containers/forms/PlanForm/index.tsx

```diff
@@ -74,6 +75,7 @@ export interface PlanFormProps {
   initialValues: PlanFormFields /** initial values for fields on the form */;
   jurisdictionLabel: string /** the label used for the jurisdiction selection */;
   redirectAfterAction: string /** the url to redirect to after form submission */;
+  JurisdictionIdToName?: typeof ConnectedJurisdictionIdToName | null;
```

Comment:
Can we add a docstring here to match all the other props?  For consistency :)

Reply:
:+1: